services/web/server/src/simcore_service_webserver/studies_dispatcher/_studies_access.py

```diff
@@ -110,7 +110,7 @@ async def _get_published_template_project(
             raise RedirectToFrontEndPageError(
                 MSG_PUBLIC_PROJECT_NOT_PUBLISHED.format(project_id=project_uuid),
                 error_code="PUBLIC_PROJECT_NOT_PUBLISHED",
-                status_code=web.HTTPNotFound.status_code,
+                status_code=web.HTTPUnauthorized.status_code,
```

Comment:
This error code is not for this purpose:  https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401
 Please do not change the code just to have a nicer message in the front-end :-) Rather change the message or the way the front-end shows https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404

Reply:
Ok, still trying to find an alternative.
Btw, I think we also use Unauthorized with the services list from the portal (http://osparc.io/view?viewer_key=simcore/services/comp/kember-cardiac-model&viewer_version=1.0.0)
![image](https://github.com/ITISFoundation/osparc-simcore/assets/18575092/13502b90-a1ac-44be-ba25-46395e4eafa5)

And that one seems a similar use-case to this.

Reply:
@pcrespov, based on a similar use-case we have (see my last comment above), you're still really against using `401: Anathorized`?